www/app/docs/auth/concepts/acclinking.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Account Linking'
+description: 'Understanding account linking in Nile Auth'
+---
+
+## Overview
+
+Account linking allows users to authenticate with multiple SSO providers to the same user.
+This is automatically enabled in Nile Auth, and will be used if you have multiple SSO providers configured 
+or if you choose to support both email/password and SSO authentication.
+
+<Tip> Combining SSO and email/password authentication requires the email address to be verified.
+You can set up [email provider and templates](/auth/email) in Nile console.</Tip>
+
+## How it works
+
+When a user first signs up, no matter what authentication method they use, they are assigned a unique user ID that is linked to their email address. 
+You can see this in the `users.users` table:
+
+```sql
+SELECT * FROM users.users;
+```
+
+If the user then signs in with a different SSO provider, and the email address is the same, the new authentication method will be linked to the same user ID.
+You can see that the user ID has multiple credentials in the `users.credentials` table:
```

Comment:
yes, it is also important to point out that the user's information will not be updated. the original method is the time we update the profile. subsequent logins do not